pkg/server/log.go

```
@@ -0,0 +1,29 @@
+package server
+
+import (
+	"io"
+	"log"
+	"strings"
+)
+
+type FilteredWriter struct {
+	io.Writer
+}
+
+var msgsToFilter = []string{
```

Wouldn't this hide real errors parsing ssl certificates from client?